Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:latest-release
```

Comment:
We have intentionally kept it as 20.04 as we do not want to break it always using the latest version, we want to test the bot in a particular version and stick to that version.